setup.py

```diff
@@ -76,6 +76,7 @@
     ],
     entry_points={
         'console_scripts': [
+            'query-generator = code_generator.main:main',
```

Comment:
Wouldn't this better fit in: https://github.com/app-sre/sretoolbox or even its own repository?
It does not directly relate to integrations.

Reply:
Fully agree. Would make sense to extract this. 

If we finally decide to use a custom code generator, then I will move the `code_generator` module to a different repo or sretoolbox. Until then I will keep it here to have the whole PoC visible in a single PR.